websites/A/AnimeSuge/metadata.json

```diff
@@ -9,8 +9,8 @@
 		"en": "AnimeSuge is a free streaming anime website that allows you to watch anime online in English subbed and dubbed.",
 		"vi_VN": "AnimeSuge là trang web phát anime miễn phí cho phép bạn xem anime với phụ đề hoặc thuyết minh Tiếng Anh."
 	},
-	"url": "animesuge.to",
-	"version": "1.1.16",
+	"url": "animesuge.net",
```

Comment:
`.to` still exists with full functionality. Use URL list or regex

Reply:
I tested it, it now works on both.